package-lock.json

```diff
@@ -5087,9 +5087,9 @@
       }
     },
     "node_modules/axios": {
-      "version": "1.6.5",
```

Comment:
I don't think I understand, the old version is already newer than the one with the fix. What am I missing? 😄